app/docs/0.10.x/loadbalancing.md

```diff
@@ -182,6 +184,116 @@ as a single target, with the specified weight. Upon every proxied request
 to this target it will query the nameserver again.
 
 
-[Back to TOC](#table-of-contents)
+[Back to Table of Contents](#table-of-contents)
+
+### **Blue-green deployments**
+
+Using the ring-balancer a blue-green deployment can be easily orchestrated for 
```

Comment:
A description (or link thereto) of what a blue-green deployment is, and how it can be useful, would be good to have here.